.github/setup-eventbrite.md

```diff
@@ -0,0 +1,124 @@
+---
+name: Scheduling courses with Eventbrite
+about: Instructions for setting up materials ready for next session. To be completed by lead instructor
+title: Scheduling courses with Eventbrite on [DATE-OR-SESSION-DESCRIPTION] for [INSTRUCTOR-NAME]
+labels: session-setup
+assignees: ''
+
+---
+
+The lead instructor needs to schedule events on [Eventbrite](https://www.eventbrite.com) using the [RSE
+Sheffield](https://www.eventbrite.co.uk/o/rse-sheffield-29660305889) organisation. If you have not already been setup as
+an administrator for this you should request to be included by another team member (see [RSEs
+Handbook](https://github.com/RSE-Sheffield/rses-handbook/blob/master/pages/eventbrite.Rmd) for further details).
+
+**NB** - The easiest approach is to copy an existing event. From the page listing events select **Past** and click on
+the three dots to the right of a past event and select `Copy Event`. Edit the title to remove `Copy of` and update the
+dates and times.
+
+
+## Basic Info
+
+The following fields should be entered into each field on the **Basic Info** page.
+
+| Field               | Value                                                     |
+|:--------------------|-----------------------------------------------------------|
+| Event Title         | Git & GitHub through GitKraken - from Zero to Hero!       |
+| Organiser           | RSE Sheffield                                             |
+| Type                | Class, Training, or Workshop                              |
+| Category            | Science & Technology                                      |
+| Tags                | git, github, gitkraken                                    |
```

Comment:
```suggestion
| Tags                | Conda, virtual environments, reproducibility                                    |
```

Reply:
Thanks for catching these and other oversights in this table, duly updated the table, apologies for skipping it.